apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ExecuteHelper.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Google.Cloud.Spanner.V1.Logging;
+
+namespace Google.Cloud.Spanner
+{
+    internal static class ExecuteHelper
+    {
+        internal static void WithErrorTranslationAndProfiling(Action t, string name)
+        {
+            try
+            {
+                Stopwatch sw = null;
+                if (Logger.LogPerformanceTraces)
```

Comment:
resharper code cleanup optimization.  let me clean this up later with another code cleanup pass.